docs/cloud-init.yml

```diff
@@ -24,10 +32,28 @@ users:
 write_files:
   - path: /tmp/bhima-mysql-user.sql
     content: |
-      CREATE USER 'bhima'@'localhost' IDENTIFIED BY 'HISCongo2013';
+      CREATE USER 'bhima'@'localhost' IDENTIFIED BY 'HISCongo2020';
```

Comment:
I don't know if it is possible now but if we can have the user `bhima` and its password in a env variable it will be very nice

Reply:
Yeah, you are right.  I kind of started that and didn't actually finish.  I'll do that.